SWLOR.Game.Server/SWLOR.Game.Server.csproj

```diff
@@ -1610,7 +1610,7 @@
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
-    <PostBuildEvent>xcopy "$(TargetDir)*.*" "C:/NeverwinterNights/NWN/mono" /E /Y /C
+    <PostBuildEvent>xcopy "$(TargetDir)*.*" "C:/NeverwinterNights/github" /E /Y /C
```

Comment:
Probably don't want to check in this change for consistency's sake. Or alternatively if you want to set it to the default location on Windows that would be ok too: %userprofile%\documents\Neverwinter Nights\mono should work ok I think.

Reply:
I have changed this to %userprofile%\documents\Neverwinter Nights (most people do not have a mono folder, unless they are testing locally).